File: tests/Generator/resource/kotlin/complex/SecurityOAuth.kt

import com.fasterxml.jackson.annotation.*

open class SecurityOAuth : Security {
    @JsonProperty("authorizationUrl")
    var authorizationUrl: String? = null
    @JsonProperty("scopes")
    var scopes: ArrayList<String>? = null
    @JsonProperty("tokenUrl")
    var tokenUrl: String? = null
}

File: tests/Generator/resource/python/complex/security_o_auth.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler, Tag
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, Annotated, Union
from .security import Security


class SecurityOAuth(Security):
    authorization_url: Optional[str] = Field(default=None, alias="authorizationUrl")
    scopes: Optional[List[str]] = Field(default=None, alias="scopes")
    token_url: Optional[str] = Field(default=None, alias="tokenUrl")
    pass


